### DISCUSSION TITLE
Excalidraw for RPG

### DISCUSSION THREAD

**Discussion by garthh** [excalidraw/excalidraw#5323] [General]
Hi !

Since a year, I'm looking for an open source whiteboard to play roleplaying games on line. I've tried Spacedeck and also Stories (non-free), and I've recently found Excalidraw which come with functionalities I seek (save and load board from disk, collaboration, hand draw, pictures...).

I've a question, anyone as already look to build a dice roller extension for Excalidraw ? 3D dices or random picture display... 

Bye

**Comment by dwelle** [comment]
Hi, Sounds cool!

> I've a question, anyone as already look to build a dice roller extension for Excalidraw ? 3D dices or random picture display...

Nope. I assume you will be using the Exaclidraw npm package? Shouldn't be hard to build the dice roller outside of the excalidraw component itself.
